Makefile

```diff
@@ -1,3 +1,13 @@
+.PHONY: all
```

Comment:
Shouldn't the `.PHONY` be `local` instead of `all`?

Reply:
Yes, but it's also better to have all phony tasks under one `.PHONY`, so I did that, too.